x-pack/test/detection_engine_api_integration/security_and_spaces/rule_execution_logic/saved_query.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { ALERT_WORKFLOW_STATUS } from '@kbn/rule-data-utils';
+import { flattenWithPrefix } from '@kbn/securitysolution-rules';
+
+import { SavedQueryCreateSchema } from '@kbn/security-solution-plugin/common/detection_engine/schemas/request';
+import {
+  ALERT_ANCESTORS,
+  ALERT_DEPTH,
+  ALERT_ORIGINAL_TIME,
+  ALERT_ORIGINAL_EVENT,
+} from '@kbn/security-solution-plugin/common/field_maps/field_names';
+import {
+  createRule,
+  deleteAllAlerts,
+  deleteSignalsIndex,
+  getOpenSignals,
+  getRuleForSignalTesting,
+} from '../../utils';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+
+/**
+ * Specific _id to use for some of the tests. If the archiver changes and you see errors
+ * here, update this to a new value of a chosen auditbeat record and update the tests values.
+ */
+const ID = 'BhbXBmkBR346wHgn4PeZ';
```

Comment:
It's the same, but by coincidence rather than needing the value to be the same across the different files. I think having it defined per file helps avoid coupling tons of different tests together so it's easier to modify specific tests when needed.